bin/run-db-upload.py

```diff
@@ -63,6 +85,18 @@ def upload_db_data(db_data):
     except Boto3Error:
         return f"ERROR: Failed to upload the new database info file: {db_data}"
 
```

Comment:
Do we want anything around the `if not s3` line above, so that in the future if we remove the AWS credentials this won't short circuit and never make it this far? Or would changes around S3 come later?

Reply:
My idea was to do kind of a multi step deploy:

1. Get the code out
2. Turn on GCS so it uploads to both GCS and S3, let that sit for a while
3. Turn on download from GCS, let that sit
4. Come back and remove the s3 references

Reply:
@bkochendorfer @pmac Can we get this rolled out to prod? Good to do ahead of the postgres/cms tango